bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/parallacticangle/PositionAnglePanel.scala

```diff
@@ -221,15 +217,26 @@ class PositionAnglePanel[I <: SPInstObsComp with PosAngleConstraintAware,
   }
 
   /**
+    * The actual copying of a given pos angle to the data object.
+    * This is done explicitly to avoid overwriting based on minor differences in double precision and to avoid
+    * adding 180 to the position angle, which causes issues.
+    **/
+  private def setInstPosAngle(newAngleDegrees: Double): Unit = editor.foreach { e =>
+    val Precision = 0.005
+    val oldAngleDegrees = e.getDataObject.getPosAngle
+    if (Math.abs(oldAngleDegrees - newAngleDegrees) >= Precision && Math.abs(Math.abs(oldAngleDegrees - newAngleDegrees) - 180) >= Precision) {
+      e.getDataObject.setPosAngle(newAngleDegrees)
```

Comment:
Well, technically, this IS called by both the parallactic angle code AND whenever the contents of the text field are manually changed. My initial thought was that every time a keystroke is typed in the field, this is called, and it is impossible, I believe, with one numerical keystroke to increment a PA by 180 so different processing for regular PA and parallactic angle shouldn't be required.

Then I realized, though, that of course, copy-pasting could change the angle by 180 with a single key combination, so that nixes that idea.

I think maybe the best thing is to do the 180 equivalence / ignore change check IF the pos angle constraint is parallactic angle, and otherwise ignore it.